### DISCUSSION TITLE
problems with binding receiver r9mx

### DISCUSSION THREAD

**Discussion by andrey2396** [iNavFlight/inav#7661] [Q&A]
i have: radiomaster tx16s, r9m 2019, r9mx, matek f405 se, inav 3
acces firmware, External Access Mod performed , but i can't connect it. Where can be problem?

**Comment by MrD-RC** [comment]
R9MX is ACCESS only. It won’t work with a RadioMaster TX16S unless you have performed then internal access mod. 

Still, this is not an iNav issue.
